packages/rum-recorder/src/domain/rrweb/record.spec.ts

```diff
@@ -123,6 +125,72 @@ describe('record', () => {
     })
   })
 
+  it('is safe to checkout during async callbacks (new mutation observer)', (done) => {
```

Comment:
I know this is an adaptation of the old test however I am not sure to understand how the expectations relate to this test case 😕

Reply:
(see previous comment explanation)